packages/nextjs-demo/babel.config.js

```diff
@@ -1,3 +1,13 @@
 module.exports = {
   presets: ['next/babel'],
+  plugins: [
+    [
+      '@babel/plugin-transform-react-jsx',
+      {
+        throwIfNamespace: false,
+        runtime: 'automatic',
+        importSource: 'ai-jsx/next',
+      },
+    ],
+  ],
```

Comment:
jfc was this all it took. I tried to do something like this so 💀 🔫  that I couldn't get it working. 😄

Reply:
To be fair I had to implement the `jsx-runtime` stuff. Maybe it breaks on `"classic"` or smth?